### ISSUE TITLE
Input fields doesn't render in editor window using Umbraco v10.2.0

### ISSUE THREAD

**Issue by cthorsoe** [skttl/umbraco-doc-type-grid-editor#282]
I just installed the latest version of DTGE on a Umbraco project running version 10.2.0, and noticed that no input fields is being rendered in the editor window. As far as I can tell this is the case for any datatype. I've uploades a few screenshots showing an example I made. I created a simple datatype with a field with alias myField and datatype as Textstring. When inserting this component in a Grid Layout, the Input field is not rendered in the editor window.

![Doc Type Definition](https://user-images.githubusercontent.com/3206219/189885362-e6894348-c5fe-4581-9564-5b9687034ed5.PNG)
![Doc Type in Editor Window](https://user-images.githubusercontent.com/3206219/189885389-9c6a2637-376f-40f8-9cef-63b7b609572d.PNG)

**Comment by sgerme** [comment]
Hi

Following, same issue here.


What I can see in browser console :
```
Possibly unhandled rejection: {"data":"System.InvalidOperationException: The view '~/App_Plugins/DocTypeGridEditor/Render/DocTypeGridEditorPreviewer.cshtml' was not found.
```
and 
```
undefined is not an object (evaluating '$scope.contentNodeModel.variants')
```

**Comment by skttl** [comment]
Sounds like the same bug as in Contentment https://github.com/leekelleher/umbraco-contentment/issues/250

Also mentioned on the core tracker https://github.com/umbraco/Umbraco-CMS/issues/12994

**Comment by sgerme** [comment]
Adding 
```
<umb-tabbed-content content="vm.content" content-node-model="{ variants: [] }"></umb-tabbed-content>
```
Solved the issue for me.

I had to import the project into my solution to keep this change.

**Comment by bjarnef** [comment]
The changes in this PR https://github.com/umbraco/Umbraco-CMS/pull/12890/files should probably check for contentNodeModel and variants exist as `contentNodeModel` is optional in `umbTabbedContent` directive.

https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/common/directives/components/content/umbtabbedcontent.directive.js#L216

![image](https://user-images.githubusercontent.com/2919859/190389490-59979901-ba55-4987-86b9-65d0458dbe60.png)


```
show-inherit="contentNodeModel.variants.length > 1 && property.variation !== 'CultureAndSegment'"
```
https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/views/components/content/umb-tabbed-content.html#L14

https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/views/components/content/umb-tabbed-content.html#L47

**Comment by skttl** [comment]
Fixed in bddf6c4265241e7819293453ee18163a02ba0703
